flutter/flutter-provider/state-management-in-flutter-using-provider-Part2.md

```diff
@@ -0,0 +1,209 @@
+# State Management in Flutter using Provider: Part 2
+
+In previous [article](https://suresh.hashnode.dev/state-management-in-flutter-using-provider-part-1-cke8vk1cb00iny0s11lu933ch)
+we have discussed the problem of using `setState` to update the app's state in the moderate or complex app.
```

Comment:
this should not be on a different line.

Reply:
You can add one more sentence after this as "...complex app. This part will cover the solution to those problems. Let's start."